### PR TITLE
Update so we can use this on rails 5

### PR DESCRIPTION
**Pull request by tophernuts**
Let's get this working with rails 5! 🎉 

## References
kbveqvdpwla11

### CODE REVIEW

**Comment by jessevanherk** [comment]
@tophernuts should we upgrade this to use circle 2 at the same time? I see it's touching travis...

File: app/helpers/recurring_select_helper.rb [hunk 1]
```diff
@@ -114,7 +114,11 @@ def initialize(object, method, template_object, default_schedules = nil, options
       end
 
       def render
-        option_tags = add_options(recurring_options_for_select(value(object), @default_schedules, @options), @options, value(object))
+        if Rails::VERSION::STRING >= '5.2'
```
Comment (seanhealy) [inline app/helpers/recurring_select_helper.rb]: Just wanted to confirm if this is this specific to Rails `5.2+` or should we make this `5+`?

Comment (tophernuts) [reply app/helpers/recurring_select_helper.rb]: Looks like rails 5.2 uses the object directly!
https://github.com/activerecord-hackery/ransack/issues/853

**Comment by ZakaryH** [comment]
looks good to me

**Pull request merged**